block/manager.go

```diff
@@ -589,7 +588,8 @@ func (m *Manager) processNextDABlock(ctx context.Context) error {
 					// are satisfied.
 					select {
 					case <-ctx.Done():
-						return errors.WithMessage(ctx.Err(), "unable to send block to blockInCh, context done")
+						return errors.New("unable to send block to blockInCh, context done")
```

Comment:
Any reason why this is changed? Also, if we're removing it, we shouldn't keep commented code regardless.

Reply:
Added this because as we are about to use `errors` package across the project, but there already exist package, `github.com/pkg/errors`, so popping it out and using `errors` package, but in the update one, it is just displaying the error message instead of err + message.

https://pkg.go.dev/github.com/pkg/errors@v0.9.1
https://pkg.go.dev/errors